packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -70,6 +70,14 @@ function get_base_diffusers_model(model: ModelData): string {
 	return model.cardData?.base_model?.toString() ?? "fill-in-base-model";
 }
 
+function get_prompt_from_diffusers_model(model: ModelData): string {
+	return (
+		model.cardData?.widget?.[0]?.text?.toString() ??
+		model.cardData?.instance_prompt?.toString() ??
```

Comment:
why do we need `toString()` ? aren't `.text` & `.instance_prompt` already strings ? as defined https://github.com/huggingface/huggingface.js/pull/909/files#diff-4dbb69668b0377cf24081f874e42ce011491bed90939745260b6f65d689dd941